conanfile_rtc.py

```diff
@@ -27,7 +27,7 @@ def package(self):
         self.copy("*.h*", src=base + "src/shaders", dst=relative + "src/shaders")
         self.copy("*.h*", src=base + "src/utils", dst=relative + "src/utils")
 
-        self.copy("skcms.h", src=base + "third_party/skcms", dst=relative + "third_party/skcms")
+        self.copy("skcms.h", src=base + "include/third_party/skcms", dst=relative + "include/third_party/skcms")
```

Comment:
Is this for sure the only header? Should we use *.h* instead? Can this line also move up to the "include" section above.

Reply:
This is the only header in that folder, I can make it `*.h` if you want.